### DISCUSSION TITLE
Where or how can I manipulate the cursor dimensions inside quill?

### DISCUSSION THREAD

**Discussion by mark8044** [singerdmx/flutter-quill#801] [Feature Request]
I've been working on the font-size widget which is working nicely for me, but one thing I cannot figure out is how can I manipulate the cursor size?

Right now if you make the font size very big, the cursor size and the selection text box stays small as if font size was original size. 

Can anyone point me in the correct direction of how I can change the cursor size dynamically?

Thanks!

**Comment by singerdmx** [comment]
Probably has to do with line height

**Reply by mark8044** [reply]
Do you know how I can manipulate that?  any direction you can point me towards?  I was hoping to set the line height inside the drop down widget and not manipulate the cursor.dart or anything

**Reply by singerdmx** [reply]
It may be very complicated.
A segment of specific line height is treated as a new segment breaking out of original line.
Say if you change half of the line to be other size, you are actually creating a new segment.

**Reply by singerdmx** [reply]
See https://youtu.be/r502AfCX-Hw regarding segments

**Reply by mark8044** [reply]
Oh I get you.  I will leave it for the experts then.  But in the meantime I'll try playing around in the area where the font size is being set, maybe something can be done there?
